docs/Calcom.md

```diff
@@ -0,0 +1,109 @@
+# Accept Bitcoin payment for your Cal.com bookings
+
+You can now accept Bitcoin payment for all your Calcom bookings and appointments. 
+Whether you're offering consultation services or one-on-one meetings, receive Bitcoin payments today directly in your wallet—no middlemen, no platform fees, and no hidden costs.
+
+
+
+## Prerequisites:
+
+Before diving into the setup process, ensure you have the following:
+
+- [Cal.com account](https://cal.com/)
+- BTCPay Server - [self-hosted](Deployment.md) or run by a [third-party host](/Deployment/ThirdPartyHosting.md) * v2.0.6 or later.
+- [Created BTCPay Server store](CreateStore.md) with [wallet set up](WalletSetup.md)
+
+
+## Set up Cal.com with BTCPay Server
+
+Login to your cal.com account. Navigate to `Apps` > `App store` > `Payments apps`
```

Comment:
Let's ensure consistency and naming of external brands according to their brand guidelines.
```suggestion
Login to your Cal.com account. Navigate to `Apps` > `App store` > `Payments apps`
```